starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -707,17 +712,18 @@ sub check_dither {
 
     $self->{dither_acq} = $acq_dither;
     $self->{dither_guide} = $guide_dither;
+    $self->{dither_guide_y_max} = $guide_dither->{ampl_y};
```

Comment:
Seems like this would be better as an attribute of `$self->{dither_guide}`, i.e. `$self->{dither_guide}->{ampl_y_max}`.  It doesn't really belong as a top-level attribute of the Obsid class.

Reply:
Seems file.  I actually had something more like that, ran into some Perl issues and then ended up with simpler data structure.  Found out 2 hours later my issue had been an extra $, so I think I can manage this time around...